fixtures/manual/most-read.ts

```diff
@@ -1,3 +1,16 @@
+/**
+ * DO NOT EDIT THIS FILE!
+ *
+ * This file was automatically generated using the gen-fixtures.js script. Any edits
+ * you make here will be lost.
+ *
+ * If the data in these fixtures is not what you expect then
+ *
+ * 1. Update the data in 'scripts/test-data/data', and then
+ * 2. Re-run node 'scripts/test-data/gen-fixtures.js'
+ *
+ */
+
```

Comment:
Ditto. This data was manually created and lives in the /manual/ folder. Data that is autoamtically generated lives in the /generated/ folder and it is that which should not be edited